### DISCUSSION TITLE
I don’t understand how to make the configuration

### DISCUSSION THREAD

**Discussion by heiheshang** [sveltia/sveltia-cms#66] [Q&A]
I installed it locally on my computer. Launched npm run dev. The site opens in the browser and I see the inscription `The configuration file could not be retrieved. Please solve the issue and try again.` I don’t quite understand where the configuration should be?

**Comment by kyoshino** [comment]
Have you previously installed Netlify CMS or Decap CMS?

**Reply by heiheshang** [reply]
no

**Reply by kyoshino** [reply]
Okay, so please install and configure Decap CMS first as described in [README](https://github.com/sveltia/sveltia-cms#getting-started).
